roles/pulp_webserver/tasks/main.yml

```diff
@@ -17,8 +17,8 @@
   fail:
     msg: "If you give one of pulp_webserver_tls_cert and pulp_webserver_tls_key, you must also give the other."
   when:
-    - pulp_webserver_tls_cert | length or pulp_webserver_tls_key | length
-    - not pulp_webserver_tls_cert | length or not pulp_webserver_tls_key | length
+    - pulp_webserver_tls_cert is not mdellweg.filters.empty or pulp_webserver_tls_key is not mdellweg.filters.empty
```

Comment:
Nice, I really like this empty filter.

https://github.com/mdellweg/ansible_filters/blob/develop/plugins/test/tests.py#L12